src/Crate/PDO/Http/ServerPool.php

```diff
@@ -112,22 +111,18 @@ public function execute(string $query, array $parameters): CollectionInterface
 
             try {
                 $response     = $this->httpClient->request('POST', '/_sql', $options);
-                $responseBody = json_decode((string) $response->getBody(), true);
+                $responseBody = json_decode((string)$response->getBody(), true);
```

Comment:
This and https://github.com/crate/crate-pdo/pull/78/commits/46165aa574ac853d99baa95b9fae749e3c57979e#diff-eb3e3531bc32d1c04369ec6f7c16fb81R89 are the only relevant code changes. Everything else is just code formatting which can be in a separate commit.